npm/README.md

```diff
@@ -0,0 +1,65 @@
+# @openfaas/faas-cli [![NPM version](https://img.shields.io/npm/v/@openfaas/faas-cli.svg?style=flat)](https://www.npmjs.com/package/@openfaas/faas-cli) [![NPM monthly downloads](https://img.shields.io/npm/dm/@openfaas/faas-cli.svg?style=flat)](https://npmjs.org/package/@openfaas/faas-cli) [![NPM total downloads](https://img.shields.io/npm/dt/@openfaas/faas-cli.svg?style=flat)](https://npmjs.org/package/@openfaas/faas-cli)
+
+> OpenFaaS CLI
+
+## Install globally
```

Comment:
Let's just state "global" install. I can't think of a reason to use it in any other way?

Reply:
I'm good with that.